### ISSUE TITLE
Double check if we are handling `\n` in file names and add an alert when IPFS sync fails

### ISSUE THREAD

**Issue by kuzdogan** [ethereum/sourcify#1303]
#1302 reports the IPFS sync hasn't been working and upon inspecting I figured we have the `\n` character problem in a file again.

We should have handled this in #1132 but apparently, it didn't fully work.

For now I'll handle the case manually but we need to:

```[tasklist]
- [x] Double check if a file with a name with `\n` is being sanitized correctly.
- [x] Write a test case
- [x] Set an alert when the IPFS sync job fails in the CD.
```

**Comment by kuzdogan** [comment]
Fixed with https://github.com/ethereum/sourcify/commit/eb746fff46c9b7729534743de2f6fcb708ee367a

**Comment by manuelwedler** [comment]
Rafael created a general dashboard that displayed all failing jobs. So the channel also included the pgdump failures. Not sure if we wanted to see this, but I excluded them now.

There was also an issue with the alert before. The ipfs-sync cronjob started jobs under different ids, and one from two weeks ago failed which was still triggering the alert. I made the query more specific now. It only checks the last started job of ipfs-sync. Hopefully that works out now.
